.circleci/config.yml

```diff
@@ -127,8 +127,11 @@ jobs:
           template: 'Angular InstantSearch'
           app: 'angular-instantsearch'
       - build_app:
-          template: 'Autocomplete.js'
+          template: 'Autocomplete.js 0'
           app: 'autocomplete.js'
+      - build_app:
+          template: 'Autocomplete'
+          app: 'autocomplete'
```

Comment:
The ESLint version used in the template is not compatible with Node 10 so I only added the CI tests to Node 12.